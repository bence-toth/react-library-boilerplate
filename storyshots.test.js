import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  suite: 'Structural snapshots'
})
