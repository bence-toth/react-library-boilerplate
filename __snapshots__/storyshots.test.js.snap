// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Structural snapshots Hello World Default 1`] = `
<p
  className="hello world"
>
  Hello, 
  John Doe
</p>
`;
